app/models/event.rb

```diff
@@ -255,6 +255,13 @@ def intersecting_events
     room.events.where(start_time: start_time).where.not(id: id)
   end
 
+  ##
+  # Compares event start_time, end_with with current time to predict current events
+  #
+  def current?
```

Comment:
Why not also as a scope like confirmed

``
 scope :confirmed, -> { where(state: 'confirmed') }
``
http://guides.rubyonrails.org/active_record_querying.html#scopes